pkgs/development/libraries/libtorrent-rasterbar/default.nix

```diff
@@ -32,12 +38,8 @@ patches = [
     "--with-libiconv=yes"
   ];
 
-  enableParallelBuilding = true;
-
-  doCheck = false; # fails to link
-
   meta = with stdenv.lib; {
-    homepage = http://www.rasterbar.com/products/libtorrent/;
+    homepage = "http://www.rasterbar.com/products/libtorrent";
```

Comment:
Why was this changed? For me it just redirects infinitely on both versions.

Reply:
It seems this url was dead before this change was made